example-models/jobs.json

```diff
@@ -1,22 +1,21 @@
 {
     "num_jobs": 2,
-    "jobs": [
-        [
-            {
-                "name": "Software Engineer",
-                "location": [45.123, 47.232],
-                "employer": "Cisco",
-                "date": 20200405,
-                "link": "https://google.com"
-            },
-            {
-                "name": "Systems Architect",
-                "location": [70.123, 47.232],
-                "employer": "Cisco",
-                "date": 20200410,
-                "link": "https://google.com"
-            }
-        ]
-    ]
+    "jobs": {
```

Comment:
Are we changing "jobs" from an array to a json object?

Reply:
In Python, it gets treated as a dictionary. I assume that there is a way to tell JS to treat it the same way.